sample/Kmd.Logic.FileSecurity.Client.ConfigurationSample/Program.cs

```diff
@@ -58,23 +60,63 @@ private static async Task Run(AppConfiguration configuration)
+            var result = await fileSecurityClient.GetCertificate(certificateId).ConfigureAwait(false);
 
-                Log.Information("Fetching certificate details for certificate id {CertificateId} ", configuration.CertificateDetails.CertificateId);
-                var result = await fileSecurityClient.GetCertificate(certificateId).ConfigureAwait(false);
+            if (result == null)
+            {
+                Log.Error("Invalid certificate id {Id}", configuration.CertificateDetails.CertificateId);
+                return;
+            }
 
-                if (result == null)
-                {
-                    Log.Error("Invalid certificate id {Id}", configuration.CertificateDetails.CertificateId);
-                    return;
-                }
+            Console.WriteLine("Certificate ID: {0} \nCertificate Name: {1}\nSubscription ID : {2}", result.CertificateId, result.Name, result.SubscriptionId);
 
-                Console.WriteLine("Certificate ID: {0} \nCertificate Name: {1}\nSubscription ID : {2}", result.CertificateId, result.Name, result.SubscriptionId);
+            // Create a Sign Connfiguration 
+            var signConfigurationRequest = BuildSignConfigurationRequest(configuration);
+            Log.Information("Creating signconfiguration...");
+            var signConfigurationResult = await fileSecurityClient.CreateSignConfigurationPdf(signConfigurationRequest).ConfigureAwait(false);
+            if (signConfigurationResult == null)
+            {
+                Log.Error("Couldn't create signconfiguration");
+                return;
             }
+
+            Console.WriteLine(
+                "Configuration created successfully. SignConfiguration ID: {0} \nSignConfiguration Name: {1}\nSubscription ID : {2}",
+                signConfigurationResult.Id,
+                signConfigurationResult.Name,
+                signConfigurationResult.SubscriptionId);
+        }
+
+        private static SignConfigurationPdfRequestDetails BuildSignConfigurationRequest(AppConfiguration configuration)
+        {
+            var pdfPrivilege = new PdfPrivilegeModel(
+                copyAllowLevel: 1,
+                changeAllowLevel: 1,
+                allowAssembly: true,
+                allowScreenReaders: true,
+                allowFillIn: true,
+                allowModifyAnnotations: true,
+                allowCopy: true,
+                allowModifyContents: true,
+                allowDegradedPrinting: true,
+                allowPrint: true,
+                printAllowLevel: 1,
+                allowAll: true,
+                forbidAll: false);
+            return new SignConfigurationPdfRequestDetails(
+                  signConfigurationId: Guid.Empty,
+                  name: "TestSignConfiguration",
+                  ownerPassword: "TestPwd",
+                  certificateId: configuration.CertificateDetails.CertificateId,
+                  subscriptionId: Guid.Empty,
```

Comment:
Why empty? There is no subscription validation in the file security endpoint?

Reply:
While instantiating  FileSecurityClient we are using subscription id. Below in `FileSecurityOptions` subscription id is available.
`var fileSecurityClient = new FileSecurityClient(httpClient, tokenProviderFactory, configuration.FileSecurityOptions);`
Hence I was not passing the subscription id here.